hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/IntegrationTestBase.kt

```diff
@@ -271,7 +287,115 @@ data class PrisonerBuilder(
+          alertType = type,
+          alertTypeDescription = "Type Description for $type",
+          // In search all alerts are not expired and active
+          expired = false,
+          active = true,
+          dateCreated = LocalDate.now(),
+        )
+      },
+      aliases = this.aliases.map { a ->
+        Alias(
+          gender = a.gender,
+          ethnicity = a.ethnicity,
+          firstName = this.firstName,
+          middleName = null,
+          lastName = this.lastName,
+          age = null,
+          dob = LocalDate.parse(this.dateOfBirth),
+          nameType = null,
+          createDate = LocalDate.now(),
+        )
+      },
+      physicalCharacteristics = mutableListOf<PhysicalCharacteristic>().also { pcs ->
+        this.physicalCharacteristics?.hairColour?.let {
+          pcs.add(PhysicalCharacteristic("HAIR", "Hair Colour", it, null))
+        }
+        this.physicalCharacteristics?.rightEyeColour?.let {
+          pcs.add(PhysicalCharacteristic("R_EYE_C", "Right Eye Colour", it, null))
+        }
+        this.physicalCharacteristics?.leftEyeColour?.let {
+          pcs.add(PhysicalCharacteristic("L_EYE_C", "Left Eye Colour", it, null))
+        }
+        this.physicalCharacteristics?.facialHair?.let {
+          pcs.add(PhysicalCharacteristic("FACIAL_HAIR", "Facial Hair", it, null))
+        }
+        this.physicalCharacteristics?.shapeOfFace?.let {
+          pcs.add(PhysicalCharacteristic("FACE", "Shape of Face", it, null))
+        }
+        this.physicalCharacteristics?.build?.let {
+          pcs.add(PhysicalCharacteristic("BUILD", "Build", it, null))
+        }
+        this.physicalCharacteristics?.shoeSize?.let {
+          pcs.add(PhysicalCharacteristic("SHOESIZE", "Shoe Size", it.toString(), null))
+        }
+      },
+      physicalMarks = mutableListOf<PhysicalMark>().also { pms ->
+        this.physicalMarks?.tattoo?.forEach {
+          pms.add(PhysicalMark("Tattoo", null, it.bodyPart, null, it.comment, null))
+        }
+        this.physicalMarks?.mark?.forEach {
+          pms.add(PhysicalMark("Mark", null, it.bodyPart, null, it.comment, null))
+        }
+        this.physicalMarks?.other?.forEach {
+          pms.add(PhysicalMark("Other", null, it.bodyPart, null, it.comment, null))
+        }
+        this.physicalMarks?.scar?.forEach {
+          pms.add(PhysicalMark("Scar", null, it.bodyPart, null, it.comment, null))
+        }
+      },
+    ).let {
+      if (released) {
+        it.copy(
+          status = "INACTIVE OUT",
+          lastMovementTypeCode = "REL",
+          lastMovementReasonCode = "HP",
+          inOutStatus = "OUT",
+          agencyId = "OUT",
+        )
+      } else {
+        it.copy(
+          lastMovementTypeCode = "ADM",
+          lastMovementReasonCode = "I",
+        )
+      }
+    },
+  )
+
+  fun toOffenderBookingNewEndpoint(): String = GsonConfig().gson().toJson(
+    OffenderBooking(
```

Comment:
Rather than loading "/templates/booking.json" as the offender booking template we do it all in code for the new endpoint.